### PR TITLE
[error-monitoring] Allow for reporting errors to a separate repo + cron endpoint to do so

### PR DESCRIPTION
**Pull request by rcebulko**
This PR:
- allows us to use a dedicated "error-reports" repository to track issues separately from user-created issues, per request of @kristoferbaxter to reduce issue noise for the UI WG
- defines cron endpoints to automatically file issues each morning for newly detected frequent errors in the wild

### CODE REVIEW

File: error-monitoring/cron.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cron:
+- description: "File issues for errors in production"
+  url: /_cron/monitor?serviceType=production
+  schedule: every day at 7:00am
```
Comment (rcebulko) [inline error-monitoring/cron.yaml:4]: Note: times are staggered so they don't all run at once and trigger various API limits

File: error-monitoring/redacted.env [hunk 2]
```diff
@@ -1,4 +1,5 @@
 GITHUB_REPO=ampproject/amphtml
+ISSUE_REPO_NAME=amphtml
```
Comment (rcebulko) [inline error-monitoring/redacted.env:2]: When the new repo is created by Joey, this will be updated.

File: error-monitoring/src/bot.ts [hunk 3]
```diff
@@ -32,7 +32,8 @@ export class ErrorIssueBot {
   constructor(
     token: string,
     private repoOwner: string,
-    private repoName: string
+    private repoName: string,
+    private issueRepoName: string | undefined
```
Comment (danielrozenberg) [inline error-monitoring/src/bot.ts:36]: As I'm learning more and more TS, I think this should be `issueRepoName?: string`

Comment (rcebulko) [reply error-monitoring/src/bot.ts]: Fixed! I always forget about that, since `string?` is the JSDoc form that Typescript rejects.

**Pull request merged**